backend/api/prisonApi.ts

```diff
@@ -400,6 +409,12 @@ export const prisonApiFactory = (client) => {
   const getOffenderWorkHistory = (context, offenderNo, earliestEndDate) =>
     get(context, `/api/offender-activities/${offenderNo}/work-history?earliestEndDate=${earliestEndDate}`)
 
+  const getTransfers = (context, parameters: GetTransferParameters) =>
+    get(
+      context,
+      `/api/movements/transfers?agencyId=${parameters.agencyId}&fromDateTime=${parameters.fromDateTime}&toDateTime=${parameters.toDateTime}&releaseEvents=${parameters.releaseEvents}&courtEvents=${parameters.courtEvents}&transferEvents=${parameters.transferEvents}`
```

Comment:
this looks really naff - would be better to use querystring.stringify or similar to do this automatically - see https://github.com/ministryofjustice/manage-hmpps-auth-accounts/blob/main/backend/api/prisonApi.js#L30